src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -76,8 +94,8 @@ export class ClrTabs implements AfterContentInit {
   }
 
   ngAfterContentInit() {
-    if (typeof this.ifActiveService.current === 'undefined') {
-      this.tabLinkDirectives.first.activate();
+    if (typeof this.ifActiveService.current === 'undefined' && this.tabLinkDirectives[0]) {
```

Comment:
Unless we have tabs component without tabs inside them it will not be a problem. It's not a very realistic case, but I hit this issue once while testing so I decided it won't hurt to add a check that what we're trying to activate actually exists.

Reply:
I kind of like the `tabLinkDirectives.first` over using the array index. As long as it is consistent I guess.